tools/code_review_tools/treeherder.py

```diff
@@ -5,6 +5,8 @@
 import slugid
 from thclient import TreeherderClient
 
+JOBS_URL = "https://treeherder.mozilla.org/#/jobs"
+
 
 def get_job_url(task_id, run_id, **params):
```

Comment:
I've just realized that 1be0fb71b98cf7dfe659dcef33b36dda3075c818 was not fully correct, as the URL for task failures don't contain the repo name.

Maybe we should make repository and revision required parameters like for `get_revision_url`

Reply:
You mean `get_job_url` ?

Reply:
I mean, maybe repository and revision should be required for get_job_url like they are for get_revision_url

Reply:
I'll make a single function then